build/webpack.config.js

```diff
@@ -26,15 +26,15 @@ const thirdParty = path.resolve('src/third-party');
 const staticFolder = path.resolve('dist');
 
 const languages = isRelease
-    ? locales
+    ? ['en-US']
     : ['en-US']; // Only 1 language needed for dev
 
 /* eslint-disable key-spacing, require-jsdoc */
 function updateConfig(conf, language, index) {
     const config = Object.assign(conf, {
         entry: {
             annotations: ['box-annotations'],
-            preview: [`${lib}/Preview.js`],
+            preview_modern: [`${lib}/Preview.js`],
```

Comment:
What's the reasoning for the name change?

Reply:
See below, this isn't a proper patch. I vaguely remember I wanted to play around with native browser module support